great-idea/services.html

```diff
@@ -1,152 +0,0 @@
-<!doctype html>
-
```

Comment:
careful here.  I think you deleted your services.html page here while you were working on the git project?